### ISSUE TITLE
Customer unable to create competencies or update existing framework(disappears)

### ISSUE THREAD

**Issue by mparsons-ce** [cassproject/cass-editor#765]
@woodkri 
Adrienne Bryant (adrienne@cxpa.org) from Customer Experience Professionals Association has been trying to create competency Frameworks. 
https://apps.credentialengine.org/publisher/competencies/?frameworkctid=ce-78f86b38-9cfa-4ba4-b862-6c6054c1c201
She added a framework and then tried to add competencies. After saving the competency disappears. 
**Scarlett logged in as Adrienne** and can see many entries of New Framework and New Competency.
![image](https://user-images.githubusercontent.com/58825866/104043781-ecd68b80-51a1-11eb-8ec4-fae7ef6c19ae.png)

She had Adrienne try both the old and new CaSS editor with the same issue (anything added disappears from the interface). Also tried with Chrome and FireFox.

**I tried logging in as Adrienne**, and first I get: 
```
502 - Web server received an invalid response while acting as a gateway or proxy server.
There is a problem with the page you are looking for, and it cannot be displayed. When the Web server (while acting as a gateway or proxy) contacted the upstream content server, it received an invalid response from the content server.
```
I refreshed the page and then could login.
If I click one of the 'New Competency' entries, I get:
![image](https://user-images.githubusercontent.com/58825866/104043077-e85da300-51a0-11eb-8de1-5991a453fed3.png)

Using both new and old CaSS I was able to add competencies. 

Is this somehow related to Adrienne's environment? It seems strange that both Scarlett and I can log in as Adrienne and do updates that she cannot.

**Comment by woodkri** [comment]
When Adrienne is logged in, can she see the frameworks and competencies she created or are the searches returning nothing for her? It almost seems as if the key being added to the object during initial save is different than the key being used for subsequent edits but I'm not sure how/why that would be happening.

**Comment by mparsons-ce** [comment]
She doesn't see any data. 
It is so strange, since Scarlett and I can see the data when logged in as her.

**Comment by woodkri** [comment]
Is the exact same identity postMessage sent to CaSS when a user logs in as themselves as is sent when you log in as another user?

**Comment by mparsons-ce** [comment]
Yes. We log in as the user in the Accounts site and then navigate to the publisher. When we log into CaSS, the system uses the CaSS public/private key for the related org from the accounts site.

**Comment by woodkri** [comment]
This was a signature expired issue. The short term solution was for the user to sync her computer clock. The long term solution is fixing the timeOffset variable, coming soon.

**Comment by woodkri** [comment]
Closed as verified with 1.4.0.
